From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Mon, 15 May 2023 14:33:58 -0700
Subject: [PATCH] fork


diff --git a/internal/provider/sdk/resource_gitlab_project.go b/internal/provider/sdk/resource_gitlab_project.go
index 7ba71018..f010caf6 100644
--- a/internal/provider/sdk/resource_gitlab_project.go
+++ b/internal/provider/sdk/resource_gitlab_project.go
@@ -429,33 +429,34 @@ var resourceGitLabProjectSchema = map[string]*schema.Schema{
 		ValidateFunc: validation.StringInSlice(validPagesAccessLevelValues, true),
 	},
 	// The GitLab API requires that import_url is also set when mirror options are used
-	// Ref: https://gitlab.com/gitlab-org/terraform-provider-gitlab/pull/449#discussion_r549729230
+	// Ref: https://github.com/gitlabhq/terraform-provider-gitlab/pull/449#discussion_r549729230
+	// Fork: When https://github.com/pulumi/pulumi-terraform-bridge/issues/540 is closed, add these back
 	"mirror": {
-		Description:  "Enable project pull mirror.",
-		Type:         schema.TypeBool,
-		Optional:     true,
-		RequiredWith: []string{"import_url"},
+		Description: "Enable project pull mirror.",
+		Type:        schema.TypeBool,
+		Optional:    true,
+		// RequiredWith: []string{"import_url"},
 	},
 	"mirror_trigger_builds": {
-		Description:  "Enable trigger builds on pushes for a mirrored project.",
-		Type:         schema.TypeBool,
-		Optional:     true,
-		Computed:     true,
-		RequiredWith: []string{"import_url"},
+		Description: "Enable trigger builds on pushes for a mirrored project.",
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Computed:    true,
+		// RequiredWith: []string{"import_url"},
 	},
 	"mirror_overwrites_diverged_branches": {
-		Description:  "Enable overwrite diverged branches for a mirrored project.",
-		Type:         schema.TypeBool,
-		Optional:     true,
-		Computed:     true,
-		RequiredWith: []string{"import_url"},
+		Description: "Enable overwrite diverged branches for a mirrored project.",
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Computed:    true,
+		// RequiredWith: []string{"import_url"},
 	},
 	"only_mirror_protected_branches": {
-		Description:  "Enable only mirror protected branches for a mirrored project.",
-		Type:         schema.TypeBool,
-		Optional:     true,
-		Computed:     true,
-		RequiredWith: []string{"import_url"},
+		Description: "Enable only mirror protected branches for a mirrored project.",
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Computed:    true,
+		// RequiredWith: []string{"import_url"},
 	},
 	"issues_template": {
 		Description: "Sets the template for new issues in the project.",
