From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Mon, 15 May 2023 14:33:58 -0700
Subject: [PATCH] expose-provider


diff --git a/shim/shim.go b/shim/shim.go
new file mode 100644
index 00000000..79644333
--- /dev/null
+++ b/shim/shim.go
@@ -0,0 +1,17 @@
+package shim
+
+import (
+	pf "github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"gitlab.com/gitlab-org/terraform-provider-gitlab/internal/provider"
+	sdkProvider "gitlab.com/gitlab-org/terraform-provider-gitlab/internal/provider/sdk"
+)
+
+func PFProvider() pf.Provider {
+	return provider.New("")()
+}
+
+func SDKProvider() *schema.Provider {
+	return sdkProvider.New("")()
+}
